server/config.py

```diff
@@ -11,7 +11,7 @@
 # JWT
 JWT_SECRET_KEY = getenv("JWT_SECRET_KEY", 'local-secret')
 JWT_TOKEN_LOCATION = ['cookies']
-JWT_ACCESS_TOKEN_EXPIRES = 1800
+JWT_ACCESS_TOKEN_EXPIRES = datetime.datetime.utcnow()+datetime.timedelta(minutes=30)
```

Comment:
@bugatti12345 Assigned token expiry to 30 minutes. Originally 1800 was assigning tokens that were already expired.